react-charts/pie-chart/step4.md

```diff
@@ -1,29 +1,48 @@
-Now we'll add tooltips to the pie chart...
+Add a legend to the pie chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It should look like this:
 
 <pre class="file">
 &lt;ChartPie
+  constrainToVisibleArea
   data={[{ x: &#39;Cats&#39;, y: 35 }, { x: &#39;Dogs&#39;, y: 55 }, { x: &#39;Birds&#39;, y: 10 }]}
+  labels={({ datum }) =&gt; `${datum.name}: ${datum.y}%`}
   height={230}
   width={350}
 /&gt;
 </pre>
 
-3) Add the following properties to that section:
+3) <strong>Add the `padding` property to that section:</strong>
+
+In this case, give it the following values. More padding will be added to the right property to accommodate the axis labels.
 
 <pre class="file" data-target="clipboard">
-constrainToVisibleArea
-labels={({ datum }) =&gt; `${datum.x}: ${datum.y}%`}
+padding={{
+  bottom: 20,
+  left: 20,
+  right: 140,
+  top: 20
+}}
 </pre>
 
-- The `constrainToVisibleArea` property will alter the position of the tooltip so that it exactly fits within the rendered SVG
 
-4) Once the preview reloads - it should look like this:
-<img src="pie-chart/assets/tooltips.png" alt="Chart with tooltips" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+4) <strong>Add the `LegendOrientation` and `legendPosition` properties to the component.</strong>

```

Comment:
Split up the `legendOrientation` and `legendPosition` properties into individual steps. And add a step to add the `legendData`